tests/console/zypper_extend.pm

```diff
@@ -0,0 +1,99 @@
+#SUSE"s openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: This is a zypper extend regression tests. poo#51521.
+#   This test is based on https://gitlab.suse.de/ONalmpantis/scripts/blob/master/zypper_regression_test.sh.
+
+# Maintainer: Marcelo Martins <mmartins@suse.cz>
+# Tags: poo#51521
+#
+use base "consoletest";
+use repo_tools 'add_qa_head_repo';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    add_qa_head_repo;
```

Comment:
You are adding this repo to install qa_test_bash ? Couldn't be any other package from official repos ?

Reply:
ok, changed to any package from official repo.